stdlib/sql/to_test.go

```diff
@@ -3,9 +3,12 @@ package sql_test
 import (
 	"testing"
 
+	_ "github.com/mattn/go-sqlite3"
+
```

Comment:
Generally, we do not use newlines between project imports, we only separate them from the stdlib imports

Reply:
yep, changed ;-)